### DISCUSSION TITLE
eternal loop

### DISCUSSION THREAD

**Discussion by dutur1935** [aclap-dev/video-downloadhelper#1341] [Help and Q&A]
### Your question

hello, people.
System: slackware linux current
browser: firefox 128.2.0esr
VideoDownloadHelper version: 9.x
companion app: vdhcoap 2.0.19

with this configuration, VideoDownloadHelper go in eternal loop for searching companion app; both if it is installed only for user or (also is deprecated) on system.

I did get old version of VideoDownloadHelper 7.6.6 and companion app vdhcoapp 1.6.3 and I have not this loop

### Addon & CoApp version and details

Informazioni su Video DownloadHelper⊗
[Forum di supporto](moz-extension://e77771b7-92b1-4cf5-a452-e9e286ddc8fb/content/about.html?panel=about#)[Pagina principale](moz-extension://e77771b7-92b1-4cf5-a452-e9e286ddc8fb/content/about.html?panel=about#)
⊕
Video DownloadHelper 7.6.6
⊕
Piattaforma
⊕
App integrativa installata
⊕
Licenza non necessaria
[Offerta da Weh](moz-extension://e77771b7-92b1-4cf5-a452-e9e286ddc8fb/content/about.html?panel=about#)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Let me quote 2 lines from your post:

> VideoDownloadHelper version: 9.x
> Video DownloadHelper 7.6.6

Those cannot both be true.

First off, 9.x is an inadequate specification of a release of VDH.  For example, my version is 9.1.0.41 beta.  You need to quote the FULL version number.

Second, I'm assuming you pasted the second block of information from the VDH settings.  I believe that information instead of what you posted above that.  That would indicate you are actually NOT running any version of VDH 9.x but rather the woefully outdated version 7.6.6.  Your pasted information fails to mention your version of the CoApp.  That would not be surprising if you're trying to run VDH 7.6.6 with the CoApp 2.0.19 you are claiming in your opening text.  That combination is doomed to fail.  You need to either install the VDH 9.1.0.41 beta (the latest version for Firefox), or you need to regress the CoApp to a version 1.x to be compatible with the VDH 7.6.6 that you appear to really be using, contrary to your different belief.

**Reply by GrampaWildWilly** [reply]
By the way, the latest Firefox for Windows is 131.0.2, which came out earlier today.  You're running Linux, a system I know nothing about, so maybe the latest Firefox isn't the same.  Still, I'm suspicious of your version of Firefox.  You really ought to be on the latest version . . . unless there's some condition of Linux that prevents that.  Like I say, I know nothing about Linux so I could easily be wrong.

**Reply by dutur1935** [reply]
If you read correctly, 
I wrote: VideoDownloadHelper version: 9.x, that is any version of VideoDownloadHelper of series 9, with the related companion app: vdhcoap 2.0.19 go in eternal loop.

Then, i wrote that the VideoDownloadHelper version 7.6.6 with the companion app version vdhcoapp 1.6.3 run correctly.

clearer than this...

**Reply by GrampaWildWilly** [reply]
OK.  My bad.  Well then.  It seems this might be a Linux issue.  Like I said, I know nothing about Linux.  Let's hope somebody with Linux experience comes in here to help you.  In the meantime, you might search this forum for other discussions that mention Linux.  There might be something useful you will find by reading those other discussions.

**Comment by Mattmjs** [comment]
How was the companion app installed, manually or using the command in the terminal ?
Also refer to the wiki and see if it helps or not

https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#linux-all
